delivery/templates/contact-form.ftl

```diff
@@ -1,16 +1,44 @@
-<form id="${contentModel.formId_s}" action="${contentModel.url_s}" method="post">
-  <#if contentModel.title_s??>
-  <strong>${contentModel.title_s}</strong><br/>
-  </#if>
+<#if contentModel.usePlugintheme_b?? && contentModel.usePlugintheme_b>
+    <link rel="stylesheet" href="/static-assets/plugins/org/craftercms/plugin/contactform/css/contact-form.css" />
+</#if>
 
-  <#list contentModel.fields_o.item as field>
-  <label for="${contentModel.formId_s}-${field.fieldName_s}">${field.fieldLabel_s}</label><br/>
-  <input type="input" id="${contentModel.formId_s}-${field.fieldName_s}" name="${field.fieldName_s}"/><br/>
-  </#list>
+<#assign validateForm = contentModel.usePluginScript_b?? && contentModel.usePluginScript_b />
 
-  <label for="${contentModel.formId_s}-message">${contentModel.messageLabel_s}</label><br/>
-  <textarea id="${contentModel.formId_s}-message" name="message"></textarea><br/>
+<form
+    id="${contentModel.formId_s}"
+    action="${contentModel.url_s}"
+    method="post"
+    class="contact-form <#if validateForm>needs-validation</#if>"
+    <#if validateForm>novalidate</#if>
```

Comment:
Is this condition correct? You want novalidate if the model states to validate?

Reply:
yeah, that's a prop that needs to be added to disable HTML5 default validation, to allow custom validation.